diagrams/toolbox/generate-diagrams.sh

```diff
@@ -0,0 +1,39 @@
+#! /bin/sh
+######################################################################
+#
+# Run the Cloudnet TOSCA toolbox.
+#
+# Copyright (c) 2021 Inria
+#
+######################################################################
+
+# Check that the CLOUDNET_BINDIR environment variable is set correctly.
+if [ ! -f ${CLOUDNET_BINDIR}/cloudnet_rc.sh ]
+then
+  echo "The CLOUDNET_BINDIR environment variable is not set correctly!"
+  exit 1
+fi
```

Comment:
This script can be executed if the user has already installed [Cloudnet TOSCA toolbox](https://github.com/Orange-OpenSource/Cloudnet-TOSCA-toolbox), right? It would be nice to have also the installation part here in this script.